pyro_risks/datasets/queries_api.py

```diff
@@ -16,6 +22,12 @@ def call_era5land(output_path: str, year: str, month: str, day: str) -> None:
         month: str
         day: str
     """
+    file_path = os.path.join(output_path, f"era5land_{year}_{month}_{day}.nc")
+
+    if os.path.exists(file_path):
+        logger.info(f"Using cached {file_path}")
+        return
+
```

Comment:
I guess this is due to the previous PR, but it was confusing for me when the CDSApi rejected my requests. This is why I'm proposing to explain somewhere that we should add a `.env` file and maybe create a template of what that file must look like `.env.dist` or just put an example in the README

Reply:
Just updated the README!